products/kong.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Kong
+category: server-app
+tags: gateway
+permalink: /kong
+releasePolicyLink: https://docs.konghq.com/gateway/latest/support-policy/
+changelogTemplate: https://docs.konghq.com/gateway/changelog/#{{'__RELEASE_CYCLE__'|split:'.'|pop|join:'-'}}
```

Comment:
```suggestion
changelogTemplate: "https://docs.konghq.com/gateway/changelog/#{{'__LATEST__'|replace:'.',''}}0"
```

This produces links such as `https://docs.konghq.com/gateway/changelog/#$x`, where `$x` is the major version number. Something like that should work better (assuming community version are always x.y.z.0 versions).